Lib/axis.py

```diff
@@ -2192,6 +2192,8 @@ def __setitem__(self, index, value):
 
     def __setslice__(self, low, high, value):
         # Hack to prevent netCDF overflow error on 64-bit architectures
+        if(self.isUnlimited() and self.isTime() and (high >= Max32int)):
```

Comment:
I guess we can leave it in, I'm not sure. I'll let you decide. But do remove isTime please.